src/scss/global.scss

```diff
@@ -162,7 +162,7 @@ button, .button {
   }
 }
 
-p, label, input {
+p, label, input, li {
```

Comment:
I'm a bit nervous about changing this global style, as it might affect stuff elsewhere. Can we just do a targeted fix for `.status-content li`?

Reply:
Just checked, and indeed if you look at the "All"/"Mentions" text on the /notifications page, it's larger now because of this change. Unfortunately global CSS makes for unpredictable changes; I probably should not have put this in there in the first place...